prober/tcp_test.go

```diff
@@ -563,6 +563,50 @@ func TestTCPConnectionQueryResponseMatching(t *testing.T) {
 
 }
 
+func TestTCPConnectionQueryResponseByteMode(t *testing.T) {
+	ln, err := net.Listen("tcp", "localhost:0")
+	if err != nil {
+		t.Fatalf("Error listening on socket: %s", err)
+	}
+	defer ln.Close()
+
+	testCTX, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+	time.Sleep(time.Millisecond * 100)
```

Comment:
can you share on why we need to sleep 100ms here??

Reply:
I see, does this test fail without the sleep? can you run it without timeout once please??

Reply:
yes please, if we see that test is flaky, we can always add it later.